### PR TITLE
add ios default-map support overlay images

### PR DESCRIPTION
**Pull request by chocky335**
Added example to test it.
Current bugs:
* Mirrored presentation of image,
* Show image just after tap screen, swipe, scale...

### CODE REVIEW

File: ios/AirMaps/AIRMap.m [hunk 1]
```diff
@@ -104,13 +105,9 @@ - (void)insertReactSubview:(id<RCTComponent>)subview atIndex:(NSInteger)atIndex
     } else if ([subview isKindOfClass:[AIRMapCircle class]]) {
         [self addOverlay:(id<MKOverlay>)subview];
     } else if ([subview isKindOfClass:[AIRMapUrlTile class]]) {
-        ((AIRMapUrlTile *)subview).map = self;
```
Comment (jschloer) [inline ios/AirMaps/AIRMap.m]: I don't think you meant to remove these lines. Doing so would remove the ability to add child components inside Views, as well as break urlTile prop switching.

**Comment by lelandrichardson** [comment]
@jschloer I'm afraid my project restructuring caused a bunch of conflicts here. Would you mind rebasing?  Thanks!

**Comment by lelandrichardson** [comment]
whoops, i meant @chocky335

**Comment by markmssd** [comment]
Howdy @chocky335 ! Any plans on fixing conflicts of your amazing PR?

Cheers!

**Comment by rborn** [comment]
Closing as it's merged in #1359 
@chocky335  if you think there is something more to do please ping me.

**Pull request closed without merging**